logs/2023-01-07.log.md

```diff
@@ -0,0 +1,13 @@
+# Saturday, January  7, 2023 11:05 AM
+
+How did you get the ghosts to damage the castle?
+1. What i did was i ran a code in my game that made my ghost cross the bridge and deal damage to the enemy castle. I first made sure that when the ghost was about to damage the castle, i made sure that the life bar for the castle would go down. So i created a sprite called life bar.png so that it would show how much damage the enemy castle took. After i did that, i made sure that the life bar went down after the ghost hit the enemy base.
+
+
+
+Explain how you set up the health bar for the castle
+2. ( answered it in question 2)
```

Comment:
You need to state the specifics of how you set up the sprite, how you set it to be under the castle, and how you got it to "shrink"